lib/bin/watch.coffee

```diff
@@ -35,7 +35,7 @@ if fs.existsSync(dockerignorePath)
 ignoredFilesList = _.map(ignoredFilesList, (line) ->
   line.replace('.', '\\.')
 )
-ignoredFilesRegex = '/' + ignoredFilesList.join('|') + '/'
+ignoredFilesRegex = new RegExp(ignoredFilesList.join('|'))
```

Comment:
I believe you'll have to escape the special regex characters in `ignoredFilesList.join('|')`.